README.md

```diff
@@ -20,6 +20,7 @@ Lobby Server is, as the name says, the server counterpart for the Lobby Client.
 
 Steps to get it up and running:
 * Run `npm install` in the project root to install all dependencies
+* Open `config.json` in `config` folder and write your GameServer.exe path
```

Comment:
There should only be a template of the `config.json`, because you'll have to add `config.json` to `.gitignore`. So move `config.json` to `config.json.template` and add `config.json` to the `.gitignore` file. Sort of like how the settings work in the GameServer repository.